Homework19

JguyTestpro opened the pull request.

JguyTestpro committed:
Homework17

JguyTestpro committed:
Merge pull request #1 from JguyTestpro/Homework17

Homework17

JguyTestpro committed:
Homework18

JguyTestpro committed:
Merge pull request #3 from JguyTestpro/Homework18

Homework18

JguyTestpro committed:
Homework18-1

JguyTestpro committed:
Merge pull request #4 from JguyTestpro/Homework18

Homework18-1

JguyTestpro committed:
Homework19

JguyTestpro committed:
Homework 19-1

Berichicko commented:
You got this problem, please correct it. Please finish your homework.

![image](https://github.com/Test-Pro-Bootcamp/A51-QA-Automation-Framework/assets/104761482/c4601a79-ebee-4a70-b3e1-8e0b2f5698ef)

HINT

Test status: 

![Test status_en](https://github.com/Test-Pro-Bootcamp/A51-QA-Automation-Framework/assets/104761482/5fd4def3-064b-4071-b41c-65196e66f43a)
	

Test failed(yellow). If at least one test receives this status, then all its parents are marked as failed.

![Test failed_en](https://github.com/Test-Pro-Bootcamp/A51-QA-Automation-Framework/assets/104761482/e827f662-02fe-4f94-aae1-e0721d9b3498)

JguyTestpro commented:
?  This was bulletproof when I ran it and submitted it!  All Passing.  Not sure what happened.  Let me check and I will re-send the homework.

Thanks Alex
________________________________
From: Aliaksandr (Alex) Zasinets ***@***.***>
Sent: Friday, November 17, 2023 12:24 PM
To: Test-Pro-Bootcamp/A51-QA-Automation-Framework ***@***.***>
Cc: Jeremy Guy ***@***.***>; Author ***@***.***>
Subject: Re: [Test-Pro-Bootcamp/A51-QA-Automation-Framework] Homework19 (PR #156)


You got this problem, please correct it. Please finish your homework.

[image]<https://user-images.githubusercontent.com/104761482/283879887-c4601a79-ebee-4a70-b3e1-8e0b2f5698ef.png>

HINT

Test status:

[Test status_en]<https://user-images.githubusercontent.com/104761482/283880163-5fd4def3-064b-4071-b41c-65196e66f43a.png>

Test failed(yellow). If at least one test receives this status, then all its parents are marked as failed.

[Test failed_en]<https://user-images.githubusercontent.com/104761482/283880218-e827f662-02fe-4f94-aae1-e0721d9b3498.png>

—
Reply to this email directly, view it on GitHub<https://github.com/Test-Pro-Bootcamp/A51-QA-Automation-Framework/pull/156#issuecomment-1816811336>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/A57EQW2QZ2JABXYT3CBZUPDYE6MTJAVCNFSM6AAAAAA7QAQERSVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTQMJWHAYTCMZTGY>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

JguyTestpro committed:
Homework19 (rev)

Here is a running copy of the code that should work for ya!  I forgot to reset the playlist before I committed it.

JguyTestpro commented:
Please Re-run the code.  I think you will find that it'll work just fine!  If you have any questions, please let me know via Teams  or direct message.  Thanks.

JguyTestpro commented:
Please recheck the code I submitted.  I should run now without issues.

Thanks Alex!!

________________________________
From: Aliaksandr (Alex) Zasinets ***@***.***>
Sent: Friday, November 17, 2023 12:24 PM
To: Test-Pro-Bootcamp/A51-QA-Automation-Framework ***@***.***>
Cc: Jeremy Guy ***@***.***>; Author ***@***.***>
Subject: Re: [Test-Pro-Bootcamp/A51-QA-Automation-Framework] Homework19 (PR #156)


You got this problem, please correct it. Please finish your homework.

[image]<https://user-images.githubusercontent.com/104761482/283879887-c4601a79-ebee-4a70-b3e1-8e0b2f5698ef.png>

HINT

Test status:

[Test status_en]<https://user-images.githubusercontent.com/104761482/283880163-5fd4def3-064b-4071-b41c-65196e66f43a.png>

Test failed(yellow). If at least one test receives this status, then all its parents are marked as failed.

[Test failed_en]<https://user-images.githubusercontent.com/104761482/283880218-e827f662-02fe-4f94-aae1-e0721d9b3498.png>

—
Reply to this email directly, view it on GitHub<https://github.com/Test-Pro-Bootcamp/A51-QA-Automation-Framework/pull/156#issuecomment-1816811336>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/A57EQW2QZ2JABXYT3CBZUPDYE6MTJAVCNFSM6AAAAAA7QAQERSVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTQMJWHAYTCMZTGY>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

Berichicko commented:
Hello,

I have checked your homework 19, and I wanted to let you know that you're doing great! Keep up the excellent work!
Best wishes from the HW team (Alex, [aliaksandr@testpro.io](mailto:aliaksandr@testpro.io))!

![804deb64dc406a20a75da05d91a1a69e](https://user-images.githubusercontent.com/104761482/235961479-9afb2841-b657-46c9-99f8-2402f933b843.jpg)

![image](https://github.com/Test-Pro-Bootcamp/A51-QA-Automation-Framework/assets/104761482/3c94522c-54fd-4d49-a148-6a02d02011cb)

JguyTestpro committed:
Update Homework19.java
